src/tests/RocketContainer.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import RocketContainer from '../components/rockets/rocketContainer/RocketContainer';
+import store from '../redux/configureStore';
+
+it('renders correctly', () => {
+  const tree = renderer
+    .create(<Provider store={store}><RocketContainer /></Provider>)
+    .toJSON();
+  expect(tree).toMatchSnapshot();
```

Comment:
- [ ] You have written only snapshot tests kindly write unit tests also to increase the test coverage.